scripts/builder/make.py

```diff
@@ -10,9 +10,16 @@ def _run(*args: str) -> bool:
         child.communicate()
         return child.returncode == 0
 
+def _which(cmd: str) -> str:
+    """shutil.which that throws on None"""
+    result = which(cmd)
+    if result is None:
+        raise ValueError(f"Can't find {cmd}, check your PATH.")
```

Comment:
I think it's worth adding a repair recommendation to the error. For example, "Run `make configure`".

Reply:
I ran into the `which` issue when trying to package the font under NixOS, where using pip isn't really an option. So I added a line for externally managed python environments.

Reply:
Got it. All is well now, the error looks clear